### PR TITLE
moyo header 

### PR DESCRIPTION
**Pull request by YelyzavetaMolochynska**
[DEMO LINK](https://YelyzavetaMolochynska.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://YelyzavetaMolochynska.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,87 @@
...
+          alt="moyologo"
+        >
+      </a>
+      <nav class="header__menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="action"
+              class="menu__link is-active"
+            >Apple
```
Comment (tetiana-ivanchenko) [inline src/index.html:36]:
```suggestion
            <a
              href="#apple"
              class="menu__link is-active"
            >
              Apple
```

Comment (tetiana-ivanchenko) [reply src/index.html:36]: fix it everywhere

File: src/style.css [hunk 2]
```diff
@@ -1 +1,72 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  margin: 0;
+  padding: 0;
```
Comment (tetiana-ivanchenko) [inline src/style.css:4]: use * selector only for box-sizing, reset directly that element styles that you need

File: src/style.css [hunk 3]
```diff
@@ -1 +1,72 @@
...
+}
+
+body {
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.header {
+  background: #fff;
```
Comment (tetiana-ivanchenko) [inline src/style.css:13]: it's defaukt color

File: src/style.css [hunk 4]
```diff
@@ -1 +1,72 @@
...
+}
+
+.menu__item {
+  list-style: none;
+  margin-right: 20px;
+}
+
+.menu__item:last-child {
+  margin-right: 0;
+}
```
Comment (tetiana-ivanchenko) [inline src/style.css:43]:
```suggestion
}

.menu__item:not(:last-child) {
  margin-right: 20px;
}
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,72 @@
...
+  color: #000;
+  font-family: Roboto;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+.menu__link:hover {
+  color: #00acdc;
```
Comment (tetiana-ivanchenko) [inline src/style.css:55]: move repeatable colors in the variable

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,87 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+      class="header"
+    >
```
Comment (ivanchuk-yurii) [inline src/index.html:20]:
```suggestion
    <header class="header">
```
from new line only if 3 or more attributes, check in other places

File: src/index.html [hunk 7]
```diff
@@ -7,8 +7,87 @@
...
+        >
+      </a>
+      <nav class="header__menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              href="#apple"
+              class="menu__link is-active"
+            >Apple
+            </a>
```
Comment (ivanchuk-yurii) [inline src/index.html:37]:
```suggestion
            >
              Apple
            </a>
```

File: src/style.css [hunk 8]
```diff
@@ -1 +1,69 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+body {
+  margin: 0;
+  padding: 0;
```
Comment (ivanchuk-yurii) [inline src/style.css:5]: `body` doesn't have default  `padding`

File: src/style.css [hunk 9]
```diff
@@ -1 +1,69 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
```
Comment (ivanchuk-yurii) [inline src/style.css:1]: don't disable errors, fix them

File: src/style.css [hunk 10]
```diff
@@ -1 +1,69 @@
...
+  padding: 0;
+  max-width: 1200px;
+  min-width: 1024px;
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  box-sizing: border-box;
```
Comment (ivanchuk-yurii) [inline src/style.css:14]: move it to `*`

File: src/style.css [hunk 11]
```diff
@@ -1 +1,69 @@
...
+  list-style: none;
+}
+
+.menu__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.menu__link {
+  color: #000;
+  font-family: Roboto;
```
Comment (ivanchuk-yurii) [inline src/style.css:44]: move it to `*`

File: src/style.css [hunk 12]
```diff
@@ -1 +1,69 @@
-/* Styles go here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+
+body {
+  margin: 0;
+  padding: 0;
+  max-width: 1200px;
+  min-width: 1024px;
```
Comment (ivanchuk-yurii) [inline src/style.css:7]: there is no need in that

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 13]
```diff
@@ -7,8 +7,70 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="/images/logo.png" alt="moyologo">
+      </a>
+      <nav class="header__menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a href="#apple" class="menu__link is-active"
+            >
```
Comment (ivanchuk-yurii) [inline src/index.html:23]:
```suggestion
            <a href="#apple" class="menu__link is-active">
```

File: src/style.css [hunk 14]
```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  color: #000;
+  font-family: Roboto, sans-serif;
```
Comment (ivanchuk-yurii) [inline src/style.css:4]: move this into `html`

File: src/style.css [hunk 15]
```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  color: #000;
```
Comment (ivanchuk-yurii) [inline src/style.css:3]: it's default value

File: src/style.css (hunk 11 above)
Comment (ivanchuk-yurii) [reply src/style.css]: sorry, I meant `html`(

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
a few comments to fix

File: src/style.css (hunk 15 above)
Comment (YelyzavetaMolochynska) [reply src/style.css]: color isn't default value in this situation cause using for links

File: src/style.css [hunk 16]
```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  box-sizing: border-box;
```
Comment (ivanchuk-yurii) [inline src/style.css:2]: `box-sizing` should be in `*`

**Review by ivanchuk-yurii — APPROVED** [review]

**Pull request closed without merging**